From ff92175474f1b77a0e5bd01adb3c65c0c2425d11 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Iy=C3=A1n=20M=C3=A9ndez=20Veiga?= <iyan.mendezveiga@hslu.ch>
Date: Thu, 26 May 2022 21:45:11 +0200
Subject: [PATCH] Add support for numpy 1.22

numba 0.55.2 supports numpy 1.22, so no reason to stay with numpy 1.21.
numpy 1.22 fixes a few known security issues (CVE-2021-41496, CVE-2021-41495)
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 3feb936c..ef2b4369 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -47,7 +47,7 @@ include_package_data = true
 test_suite = tests
 python_requires = >=3.7
 install_requires =
-    numpy >= 1.18.4, < 1.22  # v1.18.4 is needed for the new random, v1.21 has breaking changes to ufuncs
+    numpy >= 1.18.4, < 1.23  # v1.18.4 is needed for the new random
     numba >= 0.53, < 0.56  # v0.53 needed for function signautres of CPUDispatchers
     typing_extensions  # Needed for use of Literal in type hints for Python 3.7
 
